src/main/webapp/scripts/user-dir.js

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Google LLC
```

Comment:
I think it's much clearer to use `directory` instead of `dir` in the file name, same for the html/css files